components/resources/jobExplorer/jobList/JobListComponent.tsx

```diff
@@ -99,18 +95,37 @@ const JobListComponent = () => {
       link: "././cyberSecurityAnalyst",
     },
     {
-      title: "Mobile Developer",
-      image: "/images/resources/jobExplorer/MobileDev.jpg",
+      title: "React Native Developer",
+      image: "/images/resources/jobExplorer/reactNative.png",
+      width: 100,
+      height: 100,
+      link: "././reactNativeDeveloper",
+      group: "mobile",
```

Comment:
added an optional group property so i could easily get these mobile jobs if i needed to filter for them or move them to a new section